identity-tools/developer_identity/create-dev-identity.sh

```diff
@@ -0,0 +1,256 @@
+# Copyright (c) 2019, Arm Limited and affiliates.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#!/bin/bash
```

Comment:
Fyi, this needs to be on the first line of the script.

Ref: https://unix.stackexchange.com/questions/214101/the-first-line-of-scripting/214107#214107

Reply:
Done